lib/validates_zipcode/formatter.rb

```diff
@@ -0,0 +1,50 @@
+module ValidatesZipcode
+  module Formatter
```

Comment:
I'd prefer this to be a class, and the methods to be simple instance methods. `zipcode` can be passed as state and the `.to_s` duplication moved to the initializer. 

`country_alpha2` would be also passed as state and `to_s.upcase`